src/Google.Api.Gax/CallSettings.cs

```diff
@@ -62,6 +73,65 @@ public sealed class CallSettings
             WriteOptions = WriteOptions,
             PropagationToken = PropagationToken,
             Credentials = Credentials,
+            RetrySettings = RetrySettings?.Clone(),
         };
+
+        // Merge other into this.
```

Comment:
Good point, yes, later we should check to see if allowing null and making it a static method could give less cloning.